### ISSUE TITLE
List slow to render after clearing text for list of 200 items

### ISSUE THREAD

**Issue by cmacdonnacha** [mjsolidarios/react-native-search-filter#24]
As a test I increased your sample Emails data to 200 items. When I text some text it filters quick enough, but when I clear it using the backspace on the phone's keyboard it takes a long time.

Is this more of a react native issue? Can it handle 200 items?

**Comment by mjsolidarios** [comment]
Not sure though. I haven't done performance testing for this project yet. I'm looking to this [repository](https://github.com/dowjones/react-dropdown-tree-select) for hints on how to improve this search component for react native.

**Comment by burhanahmed92** [comment]
Yes it filters items very slow. Usually takes 2-3 seconds to filter on 200-400 array list
